tools/db_crashtest.py

```diff
@@ -837,15 +838,15 @@ def execute_cmd(cmd, timeout=None):
     return hit_timeout, child.returncode, outs.decode("utf-8"), errs.decode("utf-8")
 
 
-def print_if_stderr_has_errors(stderr):
-    for line in stderr.split("\n"):
-        if line != "" and not line.startswith("WARNING"):
-            print("stderr has error message:")
-            print("***" + line + "***")
+def print_if_stderr_has_errors(stderr, print_as_stderr=False):
```

Comment:
default value can be removed if we are passing the argument at all calling places.